test/jtreg/generator/test8246341/LibTest8246341Test.java

```diff
@@ -49,27 +49,27 @@ public class LibTest8246341Test {
     @Test
     public void testPointerArray() {
         boolean[] callbackCalled = new boolean[1];
-        try (Arena arena = Arena.openConfined()) {
+        try (Arena arena = Arena.ofConfined()) {
             var callback = func$callback.allocate((argc, argv) -> {
                 callbackCalled[0] = true;
                 assertEquals(argc, 4);
-                assertEquals(argv.get(C_POINTER, 0).getUtf8String(0), "java");
-                assertEquals(argv.get(C_POINTER, C_POINTER.byteSize() * 1).getUtf8String(0), "python");
-                assertEquals(argv.get(C_POINTER, C_POINTER.byteSize() * 2).getUtf8String(0), "javascript");
-                assertEquals(argv.get(C_POINTER, C_POINTER.byteSize() * 3).getUtf8String(0), "c++");
-            }, arena.scope());
+                assertEquals(argv.getAtIndex(C_POINTER, 0).getUtf8String(0), "java");
+                assertEquals(argv.getAtIndex(C_POINTER, 1).asUnbounded().getUtf8String(0), "python");
+                assertEquals(argv.getAtIndex(C_POINTER, 2).asUnbounded().getUtf8String(0), "javascript");
+                assertEquals(argv.getAtIndex(C_POINTER, 3).asUnbounded().getUtf8String(0), "c++");
```

Comment:
Is `asUnbounded` really needed here? Looking at the layout of `C_POINTER`, it has an unbounded  target layout attached:

```
final class Constants$root {
...
    static final OfAddress C_POINTER$LAYOUT = ADDRESS.withBitAlignment(64).withTargetLayout(MemoryLayout.sequenceLayout(Constants$root.C_CHAR$LAYOUT));
}
```